src/components/Responses/ResponsesTableRow.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <td :class="$style.tableItemTitle">
+    <router-link
+      :to="'/questionnaires/' + responseSummary.questionnaireID"
+      :class="$style.link"
+    >
+      {{ responseSummary.questionnaire_title }}
+    </router-link>
+  </td>
+  <td :class="$style.tableItemDate">
+    {{
+      responseSummary.res_time_limit === null
+        ? 'なし'
+        : responseSummary.res_time_limit
+    }}
+  </td>
+  <td :class="$style.tableItemDate">
+    {{
+      (responseSummary.submitted_at === null
+        ? responseSummary.submitted_at == '未提出'
+        : getMyResponses,
+      responseSummary.submitted_at)
```

Comment:
```suggestion
      responseSummary.submitted_at === null
        ?  '未提出'
      responseSummary.submitted_at
```
ここどういう意図でやりました？
こういうことがやりたかったんですかね 🤔

Reply:
冷静に考え直したら↑の感じで良さそうだなってなりました